apps/ciphers.c

```diff
@@ -135,6 +143,20 @@ int ciphers_main(int argc, char **argv)
             min_version = TLS1_2_VERSION;
             max_version = TLS1_2_VERSION;
             break;
+#ifndef OPENSSL_NO_DTLS1
```

Comment:
I don't think these guards are actually necessary here. We don't have them for the TLS options.